addons/mail/static/src/components/composer/composer.scss

```diff
@@ -13,209 +14,70 @@
 
     &.o-has-current-partner-avatar {
         grid-template-columns: 50px 1fr;
-        padding: map-get($spacers, 3) map-get($spacers, 3) map-get($spacers, 4) map-get($spacers, 1);
-
-        &:not(.o-has-footer) {
-            padding-bottom: map-get($spacers, 4);
-        }
-
-        &:not(.o-has-header) {
-            padding-top: map-get($spacers, 4);
-
-            &.o-is-in-thread-view {
-                padding-top: map-get($spacers, 1);
-                padding-bottom: map-get($spacers, 1);
-            }
-        }
-    }
-}
-
-.o_Composer_actionButtons {
-    &.o-composer-is-compact {
-        display: flex;
-    }
-    &:not(.o-composer-is-compact) {
-        margin-top: map-get($spacers, 2);
     }
 }
 
 .o_Composer_attachmentList {
-    flex: 1 1 auto;
-
     &.o-composer-is-compact {
         max-height: 100px;
     }
 
     &:not(.o-composer-is-compact) {
-        overflow-y: auto;
         max-height: 300px;
     }
 }
 
 .o_Composer_buttons {
-    display: flex;
-    align-items: stretch;
-    align-self: stretch;
-    flex: 0 0 auto;
-    min-height: 41px; // match minimal-height of input, including border width
-
-    &:not(.o-composer-is-compact) {
-        border: 0;
-        height: auto;
-        padding: 0 10px;
-        width: 100%;
-    }
+    min-height: $o-mail-composer-text-input-height + $border-width; // match minimal-height of input, including border width
 }
 
 .o_Composer_coreFooter {
     grid-area: core-footer;
-    overflow-x: hidden;
-
-    &:not(.o-composer-is-compact) {
-        margin-left: 0;
-    }
 }
 
 .o_Composer_coreHeader {
     grid-area: core-header;
 }
 
-.o_Composer_cancelReply {
-    cursor: pointer;
-}
-
 .o_Composer_coreMain {
     grid-area: core-main;
     min-width: 0;
-    display: flex;
-    flex-wrap: nowrap;
-    align-items: flex-start;
-    flex: 1 1 auto;
-
-    &:not(.o-composer-is-compact) {
-        flex-direction: column;
-    }
 }
 
 .o_Composer_currentPartnerAvatar {
-    width: 36px;
-    height: 36px;
+    width: $o-mail-thread-avatar-size;
+    height: $o-mail-thread-avatar-size;
 }
 
-.o_Composer_followers,
-.o_Composer_suggestedPartners {
```

Comment:
I removed the `o_Composer_suggestedPartners` class because I couldn't find any element with this class, am I missing something?